package.json

```diff
@@ -4,7 +4,7 @@
   "description": "A node.js version of unix's `tail -f` command",
   "main": "index.js",
   "scripts": {
-    "test": "tap",
+    "test": "tools/test-ci.sh",
```

Comment:
That's a fair point, but I guess my thought was that they're almost the same thing, so it felt like unnecessary misdirection to have a `test:ci` and a `test`.  The only difference is that CI would need the .xml file, and locally, we would not.  

On the other hand, sometimes I use `tap -b` which might not create the output file correctly, and I wouldn't want to see the .xml generation fail locally.  I'll take the suggestion just for that reason.  Thanks.

Reply:
Also, for the record, the only reason I had to even make the bash script is because generating the .xml file from within `Jenkinsfile` was not working for me.  The command was correct, but it was failing with no error message.  The bash file felt nice by keeping everything together, and it's a common pattern we use, so I went with it rather than waste time on Jenkins.